lib/rules/at-rule-descriptor-no-unknown/__tests__/index.mjs

```diff
@@ -0,0 +1,106 @@
+import rule from '../index.mjs';
+const { messages, ruleName } = rule;
+
+testRule({
+	ruleName,
+	config: true,
+
+	accept: [
+		{
+			code: '@counter-style foo { system: cyclic; symbols: url(gold-medal.svg); suffix: " "; }',
+		},
+		{
+			code: '@property --foo { syntax: "<integer>"; inherits: false; initial-value: 0; }',
+		},
+		// TODO: The following test can be supported once mdn/data, is updated and the css-tree is updated.
+		// see: https://github.com/mdn/data/blob/6b4cae7c0aa96c199a982f2b6efeda7ce67eb515/css/at-rules.json
+		// {
+		// 	code: '@view-transition { navigation: auto; }',
+		// },
+		// {
+		// 	code: '@container (width > 400px) and (height > 400px) { a { width: 50%; } }',
+		// },
+		// {
+		// 	code: '@container not (width > 400px) { a { width: 50%; } }',
+		// },
+		// {
+		// 	code: '@container summary (min-width > 400px) { @container (min-width: 800px) { a { width: 50%; } } }',
+		// },
+	],
+
+	reject: [
+		{
+			code: '@counter-style foo { bar: 0; }',
+			message: messages.rejected('@counter-style', 'bar'),
+			line: 1,
+			column: 22,
+			endLine: 1,
+			endColumn: 29,
+		},
+		{
+			code: '@property --foo { bar: 0; }',
+			message: messages.rejected('@property', 'bar'),
+			line: 1,
+			column: 19,
+			endLine: 1,
+			endColumn: 26,
+		},
```

Comment:
Let's add a test for multiple warnings:

```css
@property --foo { bar: 0; baz: 0; }
```

And another for a descriptor that's known for one type of at-rule but not another:

```css
@property --foo { system: cyclic; }
```